blackbox/docs/sql/statements/create-snapshot.txt

```diff
@@ -83,14 +87,15 @@ Clauses
 The following configuration parameters can be used to modify how the snapshot
 is created:
 
-:wait_for_completion: (Default ``false``) By default the request returns once
-                      the snapshot creation started. If set to ``true`` the
-                      request returns after the whole snapshot was created or
-                      an error occurred. The :ref:`sys.snapshots
-                      <sys-snapshots>` table can be queried to track the
-                      snapshot creation progress if ``wait_for_completion`` has
-                      been set to ``false``.
-
-:ignore_unavailable: (Default ``false``) if a given table does not exist the
-                     command will fail by default. If set to ``true`` these
-                     tables are ignored and not included in the snapshot.
+:wait_for_completion:
+  (Default ``false``) By default the request returns once the snapshot
```

Comment:
Shouldn't we metnion the default on a separate line like here: https://github.com/crate/crate/pull/6751/files#diff-2ccef31c33f18efff672b1f6df6533aaR64

Reply:
yes we probably should, but this edit is to fix formatting only. I have tried not to edit the actual text (because that can quickly take you down a rabbit hole and end up complicating the pull request)

Reply:
ok!